docs/_static/custom.css

```
@@ -1,395 +1,738 @@
+  --pst-color-success: var(--orion-blue) !important;
+  --pst-color-info: var(--orion-blue) !important;
+  --pst-color-warning: var(--orion-navy) !important;
+  --pst-color-danger: var(--orion-navy) !important;
+  --pst-color-link: var(--orion-blue) !important;
+  --pst-color-link-hover: var(--orion-blue-hover) !important;
+  --pst-color-border: var(--orion-hairline) !important;
+  --pst-color-inline-code: var(--orion-blue) !important;
+  --pst-color-inline-code-background: var(--orion-blue-wash) !important;
+  --pst-color-table-row-hover-bg: var(--orion-blue-wash) !important;
+  --pst-color-target: transparent !important;
+  color-scheme: light !important;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)